### ISSUE TITLE
Question: What exactly is returned if interval is used?

### ISSUE THREAD

**Issue by sqlninja** [nikolalsvk/render_async#81]
The readme says that the HTML can be handled by passing in an HTML element name and HTML element class, but this doesn't explain that is being returned...

Is the content of the rendered partial being returned, and do we need to handle an event?

**Comment by nikolalsvk** [comment]
Thanks for opening the issue @sqlninja.

To answer your question - If you decide to use the interval option for `render_async`, the initial container won't be replaced like in other cases.

You will get something like this when `render_async` call is done:

```html
<div id="render_async_04229e7abe1507987376">
  ... Your response HTML
</div>
```

**In other cases**, the parent div would be **eliminated** after the request is done, BUT not when you're using the interval. Reason for this is so `render_async` can now which element to append to after each request when polling for changes.

Did this help somehow?

**Comment by sqlninja** [comment]
So does it keep reloading the rendered partial or the innerHtml of the partial?

**Comment by nikolalsvk** [comment]
It finds the div element, the parent element with the ID in the previous example and changes its innerHTML. This happens if interval is present.

https://github.com/renderedtext/render_async/blob/master/app/views/render_async/_request_vanilla.js.erb#L30

**Comment by nikolalsvk** [comment]
Hey, @sqlninja! I've just released [2.1.5 version](https://github.com/renderedtext/render_async/releases/tag/2.1.5) of this gem and it returns the associated DOM element with the request. This might be interesting for you if you're still using this gem.

Cheers!
